### PR TITLE
Update mentored sprints website

### LINKED ISSUE #5: Update website for 2020
As per <https://github.com/pycon-mentored-sprints/organisational/issues/29> we decided to update the site based on the previous year's﻿

### PR DESCRIPTION
**Pull request by trallard**
This PR request adds the following changes:
- Remove 2019 specific content
- Updates to add details for 2020
- Makes sure that the style and theme matches the new digital assets
- Updates the docs, adds licenses and contributing guidelines
﻿

### CODE REVIEW

**Comment by trallard** [comment]
Related to #5

**Comment by trallard** [comment]
Assigned people here, if someone can check and approve merge it would be awesome

File: contributor_resources.rst [hunk 1]
```diff
@@ -54,4 +54,4 @@ and the step can vary from one project to another. We recommend that you check o
 contributing guidelines of the project that you’re interested in, and pre-install the
 requirements before coming to the sprint. If you're stuck in any way, let us know so we
 can help. We have also requested each of the projects that signed up to have their
-contributing guidelines up to date by **April 26th**.
+contributing guidelines up to date by **April 3rd**.
```
Comment (Cheukting) [inline contributor_resources.rst:57]: I would also propose putting my step by step guide for 1st pull request here: https://slides.com/cheukting_ho/1st_pr

**Review by Cheukting — APPROVED** [review]
All looks good to me. I also proposed putting my step to step guide to making their 1st PR.

File: contributor_resources.rst (hunk 1 above)
Comment (trallard) [reply contributor_resources.rst:57]: Awesome @Cheukting will add this!

**Pull request merged**